### DISCUSSION TITLE
Confusing setting: Theme -> Theme options -> Table of contents position

### DISCUSSION THREAD

**Discussion by muflondev** [requarks/wiki#7489] [Error / Bug Report] [answered]
Hi,

The setting in "Theme -> Theme options -> Table of contents position" seems to control the position of the history ("last edited by") window. However, the description reads: "Select whether the table of contents is shown on the left, right or not at all."

![Screenshot from 2024-12-20 20-19-16](https://github.com/user-attachments/assets/aba03cfa-0b18-47e9-84bd-cbc0932a0f8f)
![Screenshot from 2024-12-20 20-19-27](https://github.com/user-attachments/assets/382c05e9-9eaf-4acc-a1d2-9c633a090789)

Or am I missing something?

thanks,
jed

**Comment by NGPixel** [comment, marked as answer]
Use headers in your page and the table of contents will appear. e.g.:

```
# Header 1

Some content

## Sub-header 1.1

Some content
```

**Reply by muflondev** [reply]
Ok, so the "table of contents position" should be called something like "side menu position". Because that side menu doesn't always contain TOC.

**Reply by NGPixel** [reply]
There's already a "side menu" (the blue navigation one) so calling it that would be even more confusing. That area's main purpose is to display the table of contents.
